tests/e2e-legacy/runTestForTags.mjs

```diff
@@ -0,0 +1,27 @@
+import { join, resolve } from "path";
+import { spawn } from "child_process";
+import { getDirName } from "./getDirName.mjs";
+
+const __dirname = getDirName();
+const ROOT = resolve(join(__dirname, "..", ".."));
+
+export const runTestForTags = (tagsToTest) => {
+  if (tagsToTest.length === 0) {
+    console.info("No clients with integration test cases have changed.");
```

Comment:
should say "with e2e-legacy test cases"

Reply:
Updated in [`aba0caf` (#6538)](https://github.com/aws/aws-sdk-js-v3/pull/6538/commits/aba0caf33a1981316cd3550c944687c8f453eb09)